ext/liquid_c/context.c

```diff
@@ -131,6 +132,27 @@ VALUE context_filtering_p(VALUE self)
     return liquid_vm_filtering(self) ? Qtrue : Qfalse;
 }
 
+bool context_init_document_body(VALUE self)
```

Comment:
This makes it seem like it is referring to the `Liquid::Context` but it is actually referring to the `Liquid::ParseContext`.  To avoid confusion, we should use `parse_context` instead of shortening it to `context` in names referring to the `ParseContext` and we should also use a separate C file if we want a place for C code related to the parse context.